R-benchmark-25.R

```diff
@@ -0,0 +1,385 @@
+# R Benchmark 2.5 (06/2008) [Simon Urbanek]
+# version 2.5: scaled to get roughly 1s per test, R 2.7.0 @ 2.6GHz Mac Pro
+# R Benchmark 2.4 (06/2008) [Simon Urbanek]
+# version 2.4 adapted to more recent Matrix package
+# R Benchmark 2.3 (21 April 2004)
+# Warning: changes are not carefully checked yet!
+# version 2.3 adapted to R 1.9.0
+# Many thanks to Douglas Bates (bates@stat.wisc.edu) for improvements!
+# version 2.2 adapted to R 1.8.0
+# version 2.1 adapted to R 1.7.0
+# version 2, scaled to get 1 +/- 0.1 sec with R 1.6.2
+# using the standard ATLAS library (Rblas.dll)
+# on a Pentium IV 1.6 Ghz with 1 Gb Ram on Win XP pro
+
+# revised and optimized for R v. 1.5.x, 8 June 2002
+# Requires additionnal libraries: Matrix, SuppDists
+# Author : Philippe Grosjean
+# eMail  : phgrosjean@sciviews.org
+# Web    : http://www.sciviews.org
+# License: GPL 2 or above at your convenience (see: http://www.gnu.org)
+#
```

Comment:
This repository is licensed MIT, but this code is GPL2

Reply:
If you use GPL code, the derived code also has to be licensed under the GPL (hence it's sometimes called "viral"): https://www.newmediarights.org/open_source/new_media_rights_open_source_licensing_guide

We typically use the MIT license (or similar more permissive licenses) to not restrict any downstream usage.